src/blocks/recommendations/README.md

```diff
@@ -0,0 +1,66 @@
+# Parse.ly Recommendations Block
+
+This block is designed to showcase links to content on your site as provided by the [Parse.ly `/related` API endpoint](https://www.parse.ly/help/api/recommendations#get-related).
```

Comment:
We should link to documentation for the related proxy endpoint here as well as the backend API.

Reply:
Now tracked in #702.